backend/corpora/dataset_processing/h5ad_data_file.py

```diff
@@ -146,7 +140,7 @@ def generate_cxg_metadata(self, convert_anndata_colors_to_cxg_colors):
 
         cxg_group_metadata = {
             "cxg_version": CxgConstants.CXG_VERSION,
-            "cxg_properties": json.dumps({"title": self.dataset_title, "about": self.dataset_about}),
```

Comment:
unused, remove from here too?

Reply:
Is `about` unused in explorer?  The current CXG 0.2.0 spec does [include it](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs/cxg.md#cxg_properties-field) still, so would leave in the property for now, even if it has no value.

Reply:
It is currently unused but will be used by the new info drawer that the CC team is working on now.

Reply:
[Asked Brian about `about` on Slack](https://czi-sci.slack.com/archives/C023Q1APASK/p1633441275138600), in case CC is expecting to it have a value.